### ISSUE TITLE
migration from babel-eslint to @babel/eslint-parser

### ISSUE THREAD

**Issue by abhijithvijayan** [abhijithvijayan/eslint-config#8]
`babel-eslint` is now deprecated in favour of https://www.npmjs.com/package/@babel/eslint-parser

**Comment by abhijithvijayan** [comment]
Migration:

```
yarn remove babel-eslint

# bump config version & install new dependencies
npx install-peerdeps @abhijithvijayan/eslint-config --dev --yarn
```

NOTE: **if you are overriding parser options**, set `requireConfigFile` to `false`

```
{
        ...
	"parserOptions": {
		"requireConfigFile": false // allow @babel/eslint-parser to run on files that do not have a Babel configuration associated with them
	},
}
```

**Comment by abhijithvijayan** [comment]
fixed in `v2.6.1`
